docs/wallets/spv.md

```diff
@@ -0,0 +1,60 @@
+
+When an SPV wallet initialises it will connect to the Decred network using peer-to-peer connections, and it will download the full set of headers and filters. It will then validate the header chain to ensure that the chain and its proof-of-work are valid. Once this is complete, the wallet will use the filters to locally identify which blocks contains owned transactions without uploading any private data to remote nodes. The wallet can then use the peer-to-peer network to download these blocks, scan them for relevant transactions and select these to update personal transaction history and balance. 
+
+
+## How is this different from a "light" wallet?
+
+"Light" wallets such as [Exodus](https://www.exodus.io/) or [Atomic](https://atomicwallet.io/) already bring some of the benefits of an SPV wallet - for example these wallets allow sending and receiving Decred transactions with a minimal start-up time and without downloading the full chain. Light wallets certainly provide a level of convenience but there are often subtle consequences because they depend upon an API provided by a centralised service:
+
+- Light wallets depend on a remote service to watch owned addresses and provide notifications when transactions are received by the addresses. Notifications could be missed or incorrect.
+
+- Users will often be required to upload an extended public key to the central server so it is aware of all of owned addresses, potentially compromising the users privacy.
+
+- Light wallets do not validate the information they receive by checking the blockchain directly - they have to blindly trust the information provided by the central server.
+
+These concerns do not apply to SPV wallets because they connect directly to the decentralised peer-to-peer Decred network, they upload no private data to remote nodes, and they validate their own transactions by inspecting the blockchain directly.
+
+
+## Are there any disadvantages?
+
+- SPV does not support voting wallets. Voting wallets have the responsibility to vote on the validity of the last block, and a wallet cannot be sure of the validity unless it fully validates the whole blockchain leading up to that block. It is possible however to purchase tickets and allocate the voting rights to a [Voting Service Provider](../mining/how-to-stake.md#pos-using-a-voting-service-provider-vsp).
+
+- SPV wallets only download blocks which have transactions related to their owned addresses, which could potentially reveal more information about the wallet than if it downloaded every single block. This only presents a very minor decrease in privacy, but it is a decrease nonetheless. This is mitigated by downloading blocks from multiple peers so no single peer is able to see the full list of blocks downloaded by a wallet. Even if a passive observer on the network is able to see which blocks are downloaded by a wallet, they are not able to identify which transactions in those blocks are relevant.
```

Comment:
The current SPV implementation is multi-peer, but it requests relevant blocks from every connected peer rather than splitting these requests up among various distinct peers.  This can be changed, but users should not be mislead into believing this mitigation is currently being performed.

Reply:
So just to clarify, if a wallet is connected to only one peer, it will download every block from that peer?

Reply:
The ones that match the filter, but yes.